packages/azure-services/src/register-azure-services-to-container.ts

```diff
@@ -79,9 +78,16 @@ export function registerAzureServicesToContainer(container: Container, credentia
 
 function setupBlobServiceClientProvider(container: interfaces.Container): void {
     IoC.setupSingletonProvider<BlobServiceClient>(iocTypeNames.BlobServiceClientProvider, container, async context => {
-        const secretProvider = context.container.get(SecretProvider);
-        const accountName = await secretProvider.getSecret(secretNames.storageAccountName);
-        const accountKey = await secretProvider.getSecret(secretNames.storageAccountKey);
+        let accountName: string;
+        let accountKey: string;
+        if (process.env.AZURE_STORAGE_NAME !== undefined && process.env.AZURE_STORAGE_KEY !== undefined) {
+            accountName = process.env.AZURE_STORAGE_NAME;
+            accountKey = process.env.AZURE_STORAGE_KEY;
+        } else {
+            const secretProvider = context.container.get(SecretProvider);
+            accountName = await secretProvider.getSecret(secretNames.storageAccountName);
+            accountKey = await secretProvider.getSecret(secretNames.storageAccountKey);
+        }
```

Comment:
This block seems to be glue code to me, setupSingletonQueueServiceURLProvider() method is also doing the same thing. Can we move this to a common place?